data/mods/Magiclysm/Spells/attunements/Void_Mage.json

```diff
@@ -0,0 +1,229 @@
+[
+  {
+    "type": "SPELL",
+    "id": "voidmage_final_grasp",
+    "name": "Final Grasp",
+    "description": "In the end, everything succumbs to the grasp of nonexistence and stasis.  This spell gives the target a foretaste of that.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SILENT", "SOMATIC", "IGNORE_WALLS", "NO_LEGS", "MUST_HAVE_CLASS_TO_LEARN" ],
+    "min_damage": 0,
+    "max_damage": 0,
+    "min_range": 5,
+    "max_range": 20,
+    "range_increment": 0.4,
+    "damage_type": "pure",
+    "min_dot": 5,
+    "max_dot": 55,
+    "dot_increment": 1.25,
+    "max_level": 40,
+    "min_duration": 500,
+    "max_duration": 5000,
+    "duration_increment": 115,
+    "spell_class": "VOID_MAGE",
+    "base_casting_time": 200,
+    "energy_source": "MANA",
+    "base_energy_cost": 300,
+    "difficulty": 9,
+    "extra_effects": [ { "id": "voidmage_final_grasp_slowdown" } ]
+  },
+  {
+    "id": "voidmage_final_grasp_slowdown",
+    "type": "SPELL",
+    "name": "Final Grasp slowdown",
+    "description": "Removes some moves from an enemy.  Subspell; you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "hostile" ],
+    "effect": "mod_moves",
+    "shape": "blast",
+    "max_level": 40,
+    "min_damage": -100,
+    "max_damage": -1000,
+    "damage_increment": -23
+  },
+  {
+    "type": "SPELL",
+    "id": "voidmage_horizon_vortex",
+    "name": "Horizon Vortex",
+    "description": "Creates a chaotic energy field, that shutter enemies inside it. Its pretty random, and can either kill everyone in area or produce almost nothing.",
```

Comment:
Again, I seem to recall having rewritten this one before?

Reply:
```diff
-    "description": "Creates a chaotic energy field, that shutter enemies inside it.  Its pretty random, and can either kill everyone in area or produce almost nothing.",
+    "description": "Creates a chaotic energy field that disrupts enemies inside it.  It is rather random, and its effects can range from almost nothing to killing all hostiles in the area.",
```
You mean this?